cmd/access/node_builder/access_node_builder.go

```diff
@@ -525,6 +529,12 @@ func (builder *FlowAccessNodeBuilder) BuildExecutionDataRequester() *FlowAccessN
 
 type Option func(*AccessNodeConfig)
 
+func WithFlowNodeBuilder(builder *cmd.FlowNodeBuilder) func(config *AccessNodeConfig) {
```

Comment:
These options aren't used anymore. I opened https://github.com/onflow/flow-go/pull/3620 to remove them which should simplify this setup